From e3b4a08c4e8b4e59d8f2fb79df8fab7883ce9b35 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 5 Mar 2016 22:24:15 +0100
Subject: [PATCH 2/3] hardcode boost_python3 library - detection wont't work
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas MÃ¼ller <schnitzeltony@googlemail.com>
---
 qt/python.pri | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/qt/python.pri b/qt/python.pri
index 550383c..d51935a 100644
--- a/qt/python.pri
+++ b/qt/python.pri
@@ -38,14 +38,8 @@ linux {
         error("Could not find ldconfig!")
     }
 
-    # Check for different boost::python naming schemes
-    LDCONFIG_OUT = $$system($$LDCONFIG_BIN -p|grep python)
-    for (b, $$list(boost_python-py34 boost_python3)) {
-        contains(LDCONFIG_OUT, "lib$${b}.so") {
-            LIBS += "-l$$b"
-            GOT_BOOST_PYTHON = True
-        }
-    }
+    LIBS += "-lboost_python3"
+    GOT_BOOST_PYTHON = True
 
     # If we couldn't find boost::python, exit with an error.
     isEmpty(GOT_BOOST_PYTHON) {
-- 
2.5.0

